src/utils/test.ts

```diff
@@ -60,6 +61,18 @@ export const createTestMinimalApp = async (
   process.currentDir = process.projectDir
 }
 
+export const resetTestAppAndReuse = async (appName: string) => {
```

Comment:
I'd probably call this `resetTestApp` since that is what it is doing.

Reply:
:), I named it `resetTestApp` but it also sets `process.projectDir` and `process.CurrentDir` as `createTestApp` does.